nuxeo-opencmis-tests/src/test/java/org/nuxeo/ecm/core/opencmis/impl/DummyCmisBlobProvider.java

```diff
@@ -69,6 +71,25 @@ public String getProviderId() {
             }
 
             @Override
+            public String getDigestAlgorithm() {
+                return AbstractBinaryManager.MD5_DIGEST;
+            }
+
+            @Override
+            public String getDigest() {
+                String digest = super.getDigest();
+                if (digest == null) {
+                    try {
+                        return DigestUtils.md5Hex(getStream());
+                    } catch (IOException e) {
+                        return null;
```

Comment:
If this shouldn't happen then just rethrow as a `RuntimeException`. If it's a legitimate case then please add a comment explaining when that happens.